platform/config/cloud/standard/cloud_template_user_provided.json

```diff
@@ -0,0 +1,21 @@
+{
+    "cloud": {
+        "provider": "aws",
+        "version": 1,
+        "configure": {
+            "cluster_user": "",
+            "cluster_size": "user_provided",
+            "cluster_type": "standard",
+            "region": "us-west-2",
+            "placement": "us-west-2a",
+            "axsys_node_type": "m3.large",
+            "axuser_node_type": "c3.2xlarge",
```

Comment:
are these random?

Reply:
These get overwritten by whatever is input from the user.